### PR TITLE
Adding experimental google chat plugin

### PR DESCRIPTION
**Pull request by josmo**
Plugin for the new google chat that's currently in preview. This currently follows the same templating as the slack plugin.

@bradrydzewski let me know if there's any issues with it. Unfortunately I couldn't find an svg image logo, So I tried to find a png that was good enough till svgs come around.

### CODE REVIEW

File: content/peloton/drone-google-chat/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+date: 2016-01-13T00:00:00+00:00
+title: Google chat
+author: josmo
+tags: [  notifications, chat  ]
+repo: josmo/drone-google-chat
+logo: gchat.png
```
Comment (bradrydzewski) [inline content/peloton/drone-google-chat/index.md]: we have a logo that you can use here https://github.com/drone/drone-plugin-index/blob/master/static/logos/gtalk.svg

Comment (josmo) [inline content/peloton/drone-google-chat/index.md]: This is actually different the gtalk. It's more in the vain of slack then google talk.

**Comment by josmo** [comment]
I'm making one more update to include conversations, which would be nice to have before publishing this. I should have it done in the next hour though.

File: content/peloton/drone-google-chat/index.md (hunk 1 above)
Comment (bradrydzewski) [inline content/peloton/drone-google-chat/index.md]: it is actually just the Google logo. When we don't have the product logo in SVG we just fallback to the company logo.

Comment (josmo) [inline content/peloton/drone-google-chat/index.md]: ah ok totally get it. I'll switch it then :)

**Comment by josmo** [comment]
@bradrydzewski ok just switched to the generic logo and added the room groupings. Let me know if you need anything else

**Pull request merged**